.changeset/config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.7.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
```

Comment:
I recommend using the GitHub generator, like here:
https://github.com/statelyai/xstate/blob/bc592582a8b486ca3a6277ca9a9c922590572346/.changeset/config.json#L3

This package would have to be installed for this work.